demos/background_subtraction_demo/cpp_gapi/README.md

```diff
@@ -5,7 +5,7 @@ This demo shows how to perform background subtraction using G-API.
 > **NOTE**: Only batch size of 1 is supported.
 
 ## How It Works
-The demo application expects an instance-segmentation-security-???? or trimap free background matting based on pixel-level segmentation approach model in the Intermediate Representation (IR) format.
+The demo application expects an instance-segmentation-security-???? or trimap free background matting based on pixel-level segmentation approach model in the Intermediate Representation (IR) format. Please note, that there aren't background matting models in `OMZ` collection.
```

Comment:
Just wondering if any other docs in `OMZ` refer to Open Model Zoo as to `OMZ`. Wouldn't it confuse readers? Just in case

Reply:
Done